media-libs/opencv/opencv-4.4.0.ebuild

```diff
@@ -0,0 +1,383 @@
+	cuda? ( tesseract? ( opencl ) )
+	dnnsamples? ( examples )
+	gflags? ( contrib )
+	glog? ( contrib )
+	contribcvv? ( contrib qt5 )
+	contribdnn? ( contrib )
+	contribfreetype? ( contrib )
+	contribhdf? ( contrib )
+	contribsfm? ( contrib eigen gflags glog )
+	contribxfeatures2d? ( contrib download )
+	java? ( python )
+	opengl? ( qt5 )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	tesseract? ( contrib )
+	?? ( gtk3 qt5 )"
+
+# The following logic is intrinsic in the build system, but we do not enforce
+# it on the useflags since this just blocks emerging pointlessly:
+#	openmp? ( !threads )
+
+RDEPEND="
+	app-arch/bzip2[${MULTILIB_USEDEP}]
+	dev-libs/protobuf:=[${MULTILIB_USEDEP}]
+	sys-libs/zlib[${MULTILIB_USEDEP}]
+	cuda? ( dev-util/nvidia-cuda-toolkit:0= )
+	contribhdf? ( sci-libs/hdf5:= )
+	contribfreetype? (
+		media-libs/freetype:2[${MULTILIB_USEDEP}]
+		media-libs/harfbuzz[${MULTILIB_USEDEP}]
+	)
+	ffmpeg? ( media-video/ffmpeg:0=[${MULTILIB_USEDEP}] )
+	gdal? ( sci-libs/gdal:= )
+	gflags? ( dev-cpp/gflags[${MULTILIB_USEDEP}] )
+	glog? ( dev-cpp/glog[${MULTILIB_USEDEP}] )
+	gphoto2? ( media-libs/libgphoto2[${MULTILIB_USEDEP}] )
+	gstreamer? (
+		media-libs/gstreamer:1.0[${MULTILIB_USEDEP}]
+		media-libs/gst-plugins-base:1.0[${MULTILIB_USEDEP}]
+	)
+	gtk3? (
+		dev-libs/glib:2[${MULTILIB_USEDEP}]
+		x11-libs/gtk+:3[${MULTILIB_USEDEP}]
+	)
+	ieee1394? (
+		media-libs/libdc1394[${MULTILIB_USEDEP}]
+		sys-libs/libraw1394[${MULTILIB_USEDEP}]
+	)
+	java? ( >=virtual/jre-1.6:* )
+	jpeg? ( virtual/jpeg:0[${MULTILIB_USEDEP}] )
+	lapack? ( virtual/lapack )
+	opencl? ( virtual/opencl[${MULTILIB_USEDEP}] )
+	openexr? ( media-libs/openexr[${MULTILIB_USEDEP}] )
+	opengl? (
+		virtual/opengl[${MULTILIB_USEDEP}]
+		virtual/glu[${MULTILIB_USEDEP}]
+	)
+	png? ( media-libs/libpng:0=[${MULTILIB_USEDEP}] )
+	python? ( ${PYTHON_DEPS} dev-python/numpy[${PYTHON_USEDEP}] )
+	qt5? (
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qttest:5
+		dev-qt/qtconcurrent:5
+		opengl? ( dev-qt/qtopengl:5 )
+	)
+	tesseract? ( app-text/tesseract[opencl=] )
+	threads? ( dev-cpp/tbb[${MULTILIB_USEDEP}] )
+	tiff? ( media-libs/tiff:0[${MULTILIB_USEDEP}] )
+	v4l? ( >=media-libs/libv4l-0.8.3[${MULTILIB_USEDEP}] )
+	vtk? ( sci-libs/vtk[rendering] )
+	webp? ( media-libs/libwebp[${MULTILIB_USEDEP}] )
+	xine? ( media-libs/xine-lib )"
+DEPEND="${RDEPEND}
+	eigen? ( dev-cpp/eigen:3 )
+	java? ( >=virtual/jdk-1.6 )
+	vaapi? ( x11-libs/libva )"
+BDEPEND="
+	java? ( >=virtual/jdk-1.6 )
+	virtual/pkgconfig"
+
+MULTILIB_WRAPPED_HEADERS=(
+	/usr/include/opencv4/opencv2/cvconfig.h
+)
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.4.0-disable-download.patch
+	"${FILESDIR}"/${PN}-3.4.1-cuda-add-relaxed-constexpr.patch
+	"${FILESDIR}"/${PN}-4.1.2-opencl-license.patch
+)
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+	java-pkg-opt-2_pkg_setup
+}
+
+src_prepare() {
+	cmake-utils_src_prepare
```

Comment:
Not cmake-utils anymore, just cmake (cmake-utils is a deprecated eclass). Does this work as-is?